frontend/src/components/shared/ComparisonDetails.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div id="comparison-details" class="card">
+  <div id="comparison-details" class="card p-0">
```

Comment:
It looks better with the previous code. 
Old one: 

<img width="469" alt="Screenshot 2021-09-22 at 09 48 33" src="https://user-images.githubusercontent.com/5833722/134303731-ac850490-f66e-419c-974e-16813d05359e.png">

New one: 
<img width="454" alt="Screenshot 2021-09-22 at 09 48 40" src="https://user-images.githubusercontent.com/5833722/134303738-ef9e2e57-90bf-4b69-ab9e-3738e338b749.png">